docs/src/components/ThemeSelect.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import { useThemeSwitch } from './ThemeSwitch';
+import { useStyles } from '@marigold/system';
+import unicornTheme from '@marigold/theme-unicorn';
+import b2bTheme from '@marigold/theme-b2b';
+
+export const themes = {
+  b2bTheme,
+  unicornTheme,
+};
+
+const themeSelectStyles = {
+  position: 'absolute',
+  right: b2bTheme.space.none,
```

Comment:
That's not how this works :-O Use the scaling values! :P

Reply:
I am using normal css values until we have a marigold-theme.